### PR TITLE
[full ci] VIC-Machine create compute resources test

### PR DESCRIPTION
**Pull request by rajanashok**
Adding the create compute tests for VIC-Machine.

### CODE REVIEW

File: tests/resources/Util.robot [hunk 1]
```diff
@@ -143,6 +143,7 @@ Cleanup VIC Appliance On Test Server
     Log To Console  Deleting the VCH appliance ${vch-name}
     ${output}=  Run VIC Machine Delete Command
     Run Keyword And Ignore Error  Cleanup VCH Bridge Network  ${vch-name}
+    Run  govc pool.destroy "*/Resources/*"
```
Comment (mhagen-vmware) [inline tests/resources/Util.robot]: can't do that indescriminately on the CI

**Comment by mhagen-vmware** [comment]
why did you close the old PR and create this new one?

File: tests/resources/Util.robot (hunk 1 above)
Comment (rajanashok) [inline tests/resources/Util.robot]: Good morning, sorry shud have given you the context on closing the old one. The Util change affected all the other CI test. So wanted to know the best course of action on how to do the clean up.

**Comment by mdubya66** [comment]
Is this PR still relevant?

**Comment by rajanashok** [comment]
This PR was part of the VIC-Machine tests which @mhagen-vmware was taking over so Closing this.

**Comment by mhagen-vmware** [comment]
I was only implementing the network tests, we still need these compute resource tests @rajanashok ?

**Comment by rajanashok** [comment]
@mhagen-vmware sounds good I will continue on this.

**Comment by rajanashok** [comment]
Closing this really old PR, will create a new one when I start working on this again.

**Pull request closed without merging**